src/WebJobs.Extensions.OpenAI.Kusto/KustoSearchProvider.cs

```diff
@@ -171,15 +171,27 @@ KustoConnectionStringBuilder GetKustoConnectionString(string connectionName)
 
     static string GetEmbeddingsString(ReadOnlyMemory<float> embedding)
     {
-        ReadOnlySpan<float> span = embedding.Span;
-        StringBuilder builder = new(span.Length * 10); // 9 is the longest string length of a float + 1 for comma
-
-        for (int i = 0; i < span.Length; i++)
+        StringBuilder sb = new();
+        foreach (float value in embedding.Span)
         {
-            builder.Append(span[i]).Append(',');
+            sb.Append(value.ToString());
```

Comment:
Why did you switch to using the `string` overload by calling `value.ToString()` instead of just using `value`? I'm concerned that this could be significantly less performant compared to the previous version, which appended the `float` directly.

Reply:
updated to use value directly, let me know if it is good